docker/Dockerfile.ubuntu

```diff
@@ -0,0 +1,94 @@
+ARG DEPS_IMAGE_NAME
```

Comment:
I think that we should consider skipping this container, unless we want to have an option to tinker with different targets in dev container.
Still, this is mostly an artifact of how we use it in CI, and most of those options can be controlled directly with CMake invocation inside the dev container.

The next step would be a layer that adds the tools for testing/debugging instead of this one.

Reply:
For now, I would leave it as it is and use it to create test env. When it is done, we can merge it with Dockerfile.deps.ubuntu (unless there is a part that needs to have cuda installed already).